install.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+SYSTEM=$(uname -s)
+
+if [ $SYSTEM = 'Darwin' ]
+  then
+    # if the system is darwin we are on a mac
+    wget https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh -O ./miniconda_install.sh
+    brew install ghostscript
+    brew link --overwrite ghostscript
+  else
+    # otherwise it is a linux machine
+    wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ./miniconda_install.sh
+    # assumue ubuntu
+    apt install python3-tk ghostscript
+fi
+
+if [ $(ls | grep -c miniconda3) ]
```

Comment:
I don't think you want brackets here since `grep` is a command. 

https://stackoverflow.com/questions/8934012/when-are-square-brackets-required-in-a-bash-if-statement